packages/ui/react-ui-mosaic/src/views/Tree/Tree.tsx

```diff
@@ -55,11 +52,8 @@ export const Tree = ({ id, Component = TreeItem, onOver, onDrop, items = [], deb
 };
 
 const TreeRoot = ({ items }: { items: TreeData[] }) => {
-  const { id, Component, compare } = useContainer();
-  const sortedItems = useMemo(() => {
-    return compare ? [...items].sort(compare) : items;
-  }, [items, compare]);
-  const itemsWithPreview = useItemsWithPreview({ items: sortedItems, path: id, strategy: 'layout-stable' });
+  const { id, Component } = useContainer();
+  const itemsWithPreview = useItemsWithPreview({ items, path: id, strategy: 'layout-stable' });
```

Comment:
`useItemsWithOrigin`? Worth a run through to ensure these stories still work?

Reply:
Just fixed this